### PR TITLE
Datagov/local testing

### PR DESCRIPTION
**Pull request by nickumia-reisys**
Only creating to try and spur github actions into ... action?

https://github.com/GSA/datagov-ckan-multi/issues/579

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
+name: Tests
+on: [pull_request]
+env:
+  CODE_COVERAGE_THRESHOLD_REQUIRED: 33
```
Comment (jbrown-xentity) [inline .github/workflows/test.yml]: Can we bump this to the best current level?

**Review by jbrown-xentity — DISMISSED** [review]
Looks great!

**Comment by jbrown-xentity** [comment]
I don't know if we care, but we might want to rename default branch to `main`...

File: .github/workflows/test.yml (hunk 1 above)
Comment (nickumia-reisys) [reply .github/workflows/test.yml]: Accomplished.

**Review by jbrown-xentity — APPROVED** [review]

**Pull request merged**